youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -429,20 +431,43 @@ def add(meta_list, info_list=None):
 
         filename = info['filepath']
         temp_filename = prepend_extension(filename, 'temp')
+        in_filenames = [filename]
+        tmp_filenames = []
+
+        options = ['-map_metadata', '1']
```

Comment:
You now have a crash in all cases when no chapters are available cause you reference to a metadata files that does not exist. What original metadata are you talking about?